async_upnp_client/client.py

```diff
@@ -15,19 +18,23 @@
     Sequence,
     Tuple,
     TypeVar,
+    cast,
 )
+from uuid import uuid4
 from xml.etree import ElementTree as ET
 from xml.sax.saxutils import escape
 
 import defusedxml.ElementTree as DET
 import voluptuous as vol
+from aiohttp import ClientSession
```

Comment:
The idea is to keep aiohttp separate from the client. The `UpnpRequester` is used to separate the implementation from the use. If there would be a need, one can create their own `UpnpRequester` and use for example `requests`.

To be honest, I don't think anybody actually does this. But still, this is the intent.

Reply:
I remember you mentioned the need for monkey patching. If you need to monkey patch the `ClientSession`, it is better to supply your own `UpnpRequester`. If needed, you can even use one of the current concrete `UpnpRequesters` as a base class.